.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "kibana-proxy"]
 	path = kibana-proxy
 	url = https://github.com/fabric8io/openshift-auth-proxy.git
+[submodule "lib/openshift-ansible"]
```

Comment:
wouldn't we want to instead just clone as part of the test rather than add a submodule?